test/Microsoft.Extensions.Logging.Testing.Tests/LoggedTestXunitTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Testing.xunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Extensions.Logging.Testing.Tests
+{
+    public class LoggedTestXunitTests : LoggedTest
+    {
+        public LoggedTestXunitTests(ITestOutputHelper output) : base(output)
```

Comment:
Do we still have to pass `ITestOutputHelper` around? Can ILoggerFactory be injected directly?

Reply:
Actually I don't think we can remove ITestOutputHelper. What if the user wants to initialize a different LoggerFactory by calling StartLog (to override test name for example)? That call needs the ITestOutputHelper.